production/kubernetes/agent-bare.yaml

```diff
@@ -22,26 +22,22 @@ spec:
         name: grafana-pdc-agent
     spec:
       containers:
-      - args:
-        - -i
-        - /var/run/secrets/grafana-pdc-agent/key
-        - ${SLUG}@${PDC_GATEWAY}
-        - -p
-        - "22"
-        - -o
-        - BatchMode=yes
-        - -o
-        - UserKnownHostsFile=/var/run/secrets/grafana-pdc-agent/known_hosts
-        - -o
-        - CertificateFile=/var/run/secrets/grafana-pdc-agent/cert.pub
-        - -o
-        - ServerAliveInterval=15
-        - -R
-        - "0"
-        - -vvv
+      - name: grafana-pdc-agent
+        env:
+        - name: TOKEN
+          valueFrom:
+            secretKeyRef:
+              key: token
+              name: grafana-pdc-agent
+        args:
+        - -cluster
+        - ${GCLOUD_PDC_CLUSTER}
+        - -token
+        - $(TOKEN)
```

Comment:
```suggestion
        - ${TOKEN}
```

Reply:
Is TOKEN the correct name ?

Reply:
This is actually meant to be `()` because its using the env var from the manifest, not the script. See https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/

Reply:
TIL